docs/reference/remap/functions/parse_ruby_hash.cue

```diff
@@ -0,0 +1,44 @@
+package metadata
+
+remap: functions: parse_ruby_hash: {
+	category: "Parse"
+	description: """
+		Parses the `value` as ruby hash.
+		"""
+
+	notices: [
+		"""
+			Only ruby types are returned. If you need to convert a `string` into a `timestamp`, consider the
+			[`parse_timestamp`](#parse_timestamp) function.
+			""",
+	]
+	arguments: [
+		{
+			name:        "value"
+			description: "The string representation of the ruby hash to parse."
+			required:    true
+			type: ["string"]
+		},
+	]
+	internal_failure_reasons: [
+		"`value` isn't a valid ruby hash formatted payload",
+	]
+
+	return: types: ["boolean", "float", "string", "object", "array", "null"]
+
+	examples: [
+		{
+			title: "Parse ruby hash"
+			source: #"""
+				parse_ruby_hash!("{ \"test\" => \"value\", \"testNum\" => 0.2, \"testObj\" => { \"testBool\" => true } }")
```

Comment:
Let's add a non hash-rocket key/value example in here as well.

Reply:
Also, to avoid having to escape all the strings, I'd consider using raw strings:

```ruby
parse_ruby_hash!(s'{ "test" => "value", "testNum" => 0.2, "testObj" => { "testBool" => true } }')
```

Reply:
It's also missing a `null` example (I noticed, because the `fn examples()` impl below does have it included).